Directory.Build.props

```diff
@@ -361,6 +377,9 @@
     <RefPath Condition="$(TargetGroup.EndsWith('aot'))">$(RefRootPath)$(TargetGroup.TrimEnd('t').TrimEnd('o').TrimEnd('a'))/</RefPath>
     <NetStandardRefPath>$(RefRootPath)netstandard/</NetStandardRefPath>
     <NetFxRefPath>$(RefRootPath)netfx/</NetFxRefPath>
+    <TestArchiveRoot>$(ProjectDir)artifacts/helix/</TestArchiveRoot>
+    <TestArchiveTestsDir>$(TestArchiveRoot)tests/</TestArchiveTestsDir>
```

Comment:
This property should also contain the BuildConfiguration so that wildcards are build configuration specific.